roles/database/tasks/initialize-fpsd-db.yml

```diff
@@ -43,11 +42,12 @@
     lc_ctype: en_US.UTF-8
     template: template0
 
-- name: Set database environment variables in `/etc/profile.d/fpsd-database.sh`.
+- name: Set the database environment variables in `/etc/bash.bashrc`.
   become: true
-  template:
-    src: fpsd-database-sh.j2
-    dest: /etc/profile.d/fpsd-database.sh
+  lineinfile:
+    line: export {{ item[0] }}={{ item[1] }}
+    dest: /etc/bash.bashrc
     mode: "0644"
     owner: root
     group: root
+  with_items: "{{ psql_env.items() }}"
```

Comment:
Clever, doesn't work as expected. Creates the following block in `/etc/bash.bashrc`:

```
export P=G
export f=p
export /=v
export l=o
export 5=4
```

Not what we want. Ansible has a built-in loop parameter with working with dict vars. Try this:

```
- name: Set the database environment variables in `/etc/bash.bashrc`.
  become: true
  lineinfile:
    line: 'export {{ item.key }}={{ item.value }}'
    regexp: '^export {{ item.key }}='
    dest: /etc/bash.bashrc
    mode: "0644"
    owner: root
    group: root
  with_dict: "{{ psql_env }}"
```